packages/integration-sdk-cli/src/generator/template/.github/workflows/integration-deployment.yml.hbs

```diff
@@ -0,0 +1,36 @@
+name: J1 Integration deployment
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  j1-integration-deployment:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get integration name
+        id: get-integration-name
+        uses: actions/github-script@0.9.0
```

Comment:
We may be able to update this...like a lot. `actions/github-script` is on v6 now.